doc/MiniLanguageTutorial.md

```diff
@@ -162,6 +162,48 @@ These rules guarantee that assignability is transitive.
 
 The compiler uses often uses type inference to infer the types of variables from the types of values assigned to them.  If a programmer wants the compiler to infer a different type, they should use an explicit type-casting operation to convert the value to the desired type.
 
+## Castability
```

Comment:
It would be useful to have a rationale for why cast ability is defined this way.  In particular, why does cast ability differ from assignability?  There's an implicit notion of subtyping here, so why have two different notions?

Reply:
We may wish to convert from for example `bool` to `uint`, where there is not a subtyping relation.